scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+    exit 1
+  fi
+}
+
+rhel_install() {
+  if [[ "$VERSION_ID" = "8.2" ]]; then
+    yum install --assumeyes --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "RHEL $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+oraclelinux_install() {
+  if [[ "$VERSION_ID" = "8.2" ]]; then
+    yum install --assumeyes --quiet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    curl -s -L -o /etc/yum.repos.d/scylla.repo http://downloads.scylladb.com/rpm/centos/scylla-$SCYLLA_RELEASE.repo
+    yum install --assumeyes --quiet scylla
+  else
+    echo "Oracle Linux $VERSION_ID is not supported by this installer."
+    exit 1
+  fi
+}
+
+debian_install() {
+  SCYLLA_DEBIAN_URL_PREFIX="http://downloads.scylladb.com/deb/debian"
+  if [ "$VERSION_ID" = "9" ]; then
+    apt update && apt-get install -qq apt-transport-https curl gnupg2 dirmngr
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_DEBIAN_URL_PREFIX/scylla-$SCYLLA_RELEASE-stretch.list"
+    apt update && apt-get install -qq scylla
+  elif [ "$VERSION_ID" = "10" ]; then
+    apt update && apt-get -qq install apt-transport-https curl gnupg2 dirmngr
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_DEBIAN_URL_PREFIX/scylla-$SCYLLA_RELEASE-buster.list"
+    apt update && apt-get -qq install scylla
+  else
+     echo "Debian $VERSION_ID is not supported by this installer."
+     exit 1
+  fi
+}
+
+ubuntu_install() {
+  SCYLLA_UBUNTU_URL_PREFIX="http://downloads.scylladb.com/deb/ubuntu"
+  if [ "$VERSION_ID" = "16.04" ]; then
+    apt update && apt-get install -qq curl apt-transport-https gnupg2
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_UBUNTU_URL_PREFIX/scylla-$SCYLLA_RELEASE-xenial.list"
+    apt update && apt-get install -qq scylla
+  elif [ "$VERSION_ID" = "18.04" ]; then
+    apt update && apt-get install -qq curl gnupg2
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_UBUNTU_URL_PREFIX/scylla-$SCYLLA_RELEASE-bionic.list"
+    apt update && apt-get install -qq scylla
+  elif [ "$VERSION_ID" = "20.04" ]; then
+    apt update && apt-get install -qq curl gnupg2
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5e08fbd8b5d6ec9c
+    curl -s -L -o /etc/apt/sources.list.d/scylla.list "$SCYLLA_UBUNTU_URL_PREFIX/scylla-$SCYLLA_RELEASE-focal.list"
+    apt update && apt-get install -qq scylla
```

Comment:
Looks like these are all exactly the same except for the short name, which can be obtained from /etc/os-release VERSION_CODENAME.

Reply:
I consolidated the installation steps between the different OS variants.